Utilities/DataCompression/include/DataCompression/DataDeflater.h

```diff
@@ -32,22 +32,20 @@ namespace AliceO2 {
    TODO: bit order: LSB to MSB in every byte or vice versa
  */
 
-template<
-  typename _RegType,
-  typename _TargetType,
-  //  typename BoundsCheck, // bounds check policy
-  class Codec
-  >
-class DataDeflater {
+template <typename _RegType, typename _TargetType,
+          //  typename BoundsCheck, // bounds check policy
+          class Codec>
+class DataDeflater
+{
  public:
   DataDeflater() : mBegin(nullptr), mEnd(nullptr), mCurrent(mEnd), mBitPosition(0), mCodec() {}
   ~DataDeflater() {}
-
   /**
    * Init target
    * TODO: think about other targets than a buffer
    */
-  int Init(_TargetType* buffer, int size) {
+  int Init(_TargetType* buffer, int size)
```

Comment:
Well, the coding conventions do not do any distinctions so I assume yes. If not we should specify, IMHO.